crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -725,6 +746,17 @@ type Object implements ObjectOwner {
 	The domain that a user address has explicitly configured as their default domain
 	"""
 	defaultNameServiceName: String
+	"""
+	A field on an object that can be added or removed on-the-fly,
+	only affects gas when accessed, and can store heterogenous values.
+	"""
+	dynamicField(dynamicFieldName: DynamicFieldName!): DynamicField
+	"""
+	An object stored as a dynamic field on the parent object.
+	This object is also accessible directly via its address.
```

Comment:
These doc comments read a little bit strangely to me.  They are explaining what dynamic fields, rather than what this endpoint is for (it's conceptual information not reference information).  Some amount of background information can be okay, but it's not the main point of this documentation.  How about something like:

```graphql
"""
Access a dynamic field on an object using its name.

Names are arbitrary Move values whose type have `copy`, `drop`, and `store`, and are specified 
using their type, and their BCS contents, Base64 encoded.
"""
dynamicField(name: DynamicFieldName!): DynamicField

"""
Access a dynamic object field on an object using its name.

Names are arbitrary Move values whose type have `copy`, `drop`, and `store`, and are specified 
using their type, and their BCS contents, Base64 encoded.

The value of a dynamic object field can also be accessed off-chain directly via its address (e.g. 
using `Query.object`).
"""
dynamicObjectField(name: DynamicFieldName!): DynamicField
```

Reply:
Ah fantastic, thank you, this is really good